exercises/concept/freelancer-rates/.docs/hints.md

```diff
@@ -2,19 +2,17 @@
 
 ## 1. Calculate the day rate given an hourly rate
 
+- Use the numeric operators from Lucian's Lucious Lasagna.
+
 ## 2. Calculate the month rate, given an hourly rate and a discount
 
-- There is a global built-in function to _parse_ a `string` to a fractional
-  number, ignoring non-numeric characters, such as the `%` (percent)-sign.
+- There is a global built-in function to _parse_ a `string` to a fractional number, ignoring non-numeric characters, such as the `%` (percent)-sign.
```

Comment:
A suggestion: you could consider linking to the function.

Reply:
@junedev suggested removing this task / rewriting it so it doesn't need parsing (cyclic concept dependency), and I like that idea a lot, so this will go away anyway :P :P

Reply:
👍